FutureNorthants/northants-website
# Fix Mac Solr container core build

### Testing
Assuming you already have a local working dev instance of the backend project.
1. Run **make stop** in the _backend_ directory
2. In Docker Desktop remove the solr container and the solr image.
3. Checkout this branch and run **make start** in the _backend_ directory. You should see the Solr image being pulled again and the Solr core will be built in the fresh Solr container.
4. In the backend CMS, rebuild your services and news search indexes. Carry out a search. Verify search is working.
